src/main/scala/is/hail/expr/FunctionRegistry.scala

```diff
@@ -1146,13 +1146,13 @@ object FunctionRegistry {
     """,
     "min" -> "Minimum value of interval.",
     "max" -> "Maximum value of interval, non-inclusive.")
+
   register("rnorm", { (mean: Double, sd: Double) => mean + sd * scala.util.Random.nextGaussian() },
     """
     Returns a random draw from a normal distribution with mean ``mean`` and standard deviation ``sd``. ``sd`` should be non-negative. This function is non-deterministic.
     """,
     "mean" -> "Mean value of normal distribution.",
     "sd" -> "Standard deviation of normal distribution.")
-
```

Comment:
I think this change is needed to keep all of the functions for each distribution together. Otherwise, we should put a space in between all functions.